src/helpers/sheet.ts

```diff
@@ -10,3 +11,13 @@ export function createDefaultRows(rowNumber: number): Row[] {
   }
   return rows;
 }
+
+export function isSheetNameEqual(name1: string | undefined, name2: string | undefined): boolean {
+  if (name1 === undefined || name2 === undefined) {
+    return false;
```

Comment:
is it possible to have both names undefined at the same time? what does it mean then? is it really different?